Fatras/include/ActsFatras/Kernel/detail/Interactor.hpp

```diff
@@ -66,6 +67,9 @@ struct Interactor {
   /// Initial particle state.
   Particle initialParticle;
 
+  /// Deviation factor from generated proper time limit
+  Particle::Scalar properTimeTolerance = 1e-2;
```

Comment:
Is this an absolute tolerance or a relative fraction?

What is a good default value here? My initial feeling was to set a default value that does not inhibit the step size. If the user adds a non-trivial decay module, it should be his responsibility to set this to something reasonable.

Reply:
I have no feeling about a reasonable regime for it but I'm open for suggestions

Reply:
Ok, but that does not answer my first question. The name suggest an absolute tolerance, but the usage in the code is a fractional tolerance relative to the generated life time limit. Since the life time limit is drawn from an exponential it can vary widely and so will the resulting absolute tolerances. I would prefer to use an absolute tolerance, which should probably be set to values comparable to the propagation tolerances (10-100µm?).

Reply:
I forgot about the first question, my bad. Indeed, I didn't mention that its a relative one. The issue with absolute values is that it may affect very short living particles and that's why I would like to keep it a relative one.

Reply:
The reason you have to have the tolerance in the first place is the propagation/ integration error, right? And that uses an absolute tolerance as far as I remember. So having a relative tolerance here, I expect you are still limited by the absolute tolerance in the propagator. And there is exists a reasonable lower limit to the proper time limit, i.e. anything below the typical impact parameter resolution is considered a prompt particle. This goes down to a few µm; any tolerance below that will not have a measurable impact on the fast simulation.

You could add something more complicated, but a single absolute tolerance is a) easy to understand and b) easy to implement.

Reply:
Partially propagator but mostly limited due to floating point comparisons. Since both are easy to understand, this point doesn't matter. The implementation is easy, yes, but since the comparison is for the proper time difference, the evaluation of gamma is necessary for each check when using the lab distance (or corresponding lab time). Then the absolute tolerance would vary for each particle or each step. So I don't see the benefit from it.

Reply:
That is true as well. To avoid drawing this out any longer: lets leave the relative tolerance but rename the variable (maybe `properTimeRelativeTolerance`?) and clearly explain what it means in the comment.

The current default value of 1% of the simulation limit feels a bit too loose. Should we decrease it to 1‰?

Reply:
I changed the name as suggested, the comment and the value. As I said: I have no feeling for proper values, so I'm fine with every small value mentioned.